PS C:\Users\tasit\Desktop\.my_telegram_git\my_tg_messenger> python server.py
Traceback (most recent call last):
  File "C:\Users\tasit\Desktop\.my_telegram_git\my_tg_messenger\server.py", line 9, in <module>
    socketio = SocketIO(app, async_mode=None, cors_allowed_origins="*")       
  File "C:\Users\tasit\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask_socketio\__init__.py", line 187, in __init__
    self.init_app(app, **kwargs)
    ~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\tasit\AppData\Local\Programs\Python\Python313\Lib\site-packages\flask_socketio\__init__.py", line 243, in init_app
    self.server = socketio.Server(**self.server_options)
                  ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\tasit\AppData\Local\Programs\Python\Python313\Lib\site-packages\socketio\base_server.py", line 31, in __init__
    self.eio = self._engineio_server_class()(**engineio_options)
               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\tasit\AppData\Local\Programs\Python\Python313\Lib\site-packages\engineio\base_server.py", line 70, in __init__
    self._async = importlib.import_module(
                  ~~~~~~~~~~~~~~~~~~~~~~~^
        'engineio.async_drivers.' + mode)._async
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\tasit\AppData\Local\Programs\Python\Python313\Lib\importlib\__init__.py", line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked 
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1022, in exec_module    
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "C:\Users\tasit\AppData\Local\Programs\Python\Python313\Lib\site-packages\engineio\async_drivers\eventlet.py", line 3, in <module>
    from eventlet.green.threading import Event
  File "C:\Users\tasit\AppData\Local\Programs\Python\Python313\Lib\site-packages\eventlet\green\threading.py", line 18, in <module>
    eventlet.patcher.inject(
    ~~~~~~~~~~~~~~~~~~~~~~~^
        'threading',
        ^^^^^^^^^^^^
        globals(),
        ^^^^^^^^^^
        ('_thread', thread),
        ^^^^^^^^^^^^^^^^^^^^
        ('time', time))
        ^^^^^^^^^^^^^^^
  File "C:\Users\tasit\AppData\Local\Programs\Python\Python313\Lib\site-packages\eventlet\patcher.py", line 109, in inject
    module = __import__(module_name, {}, {}, module_name.split('.')[:-1])     
  File "C:\Users\tasit\AppData\Local\Programs\Python\Python313\Lib\threading.py", line 35, in <module>
    _start_joinable_thread = _thread.start_joinable_thread
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'eventlet.green.thread' has no attribute 'start_joinable_thread'
PS C:\Users\tasit\Desktop\.my_telegram_git\my_tg_messenger> 